_posts/2021-07-01-release-1.13.2.md

```diff
@@ -0,0 +1,348 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22799'>FLINK-22799</a>] -         Missing documentation: State backwards compatibility is not support for Flink SQL
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22873'>FLINK-22873</a>] -         Add ToC to configuration documentation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22905'>FLINK-22905</a>] -         Fix missing comma in SQL example in &quot;Versioned Table&quot; page
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22939'>FLINK-22939</a>] -         Generalize JDK switch in azure setup
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22959'>FLINK-22959</a>] -         Keep the job jar files  on JobManager 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-22996'>FLINK-22996</a>] -         The description about coalesce is wrong
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23009'>FLINK-23009</a>] -         Bump up Guava in Kinesis Connector
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23052'>FLINK-23052</a>] -         cron_snapshot_deployment_maven unstable on maven
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23138'>FLINK-23138</a>] -         Raise an exception if types other than PickledBytesTypeInfo are specified for state descriptor 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23156'>FLINK-23156</a>] -         Change the reference of &#39;docs/dev/table/sql/queries&#39;
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23157'>FLINK-23157</a>] -         Fix missing comma in SQL example in &quot;Versioned View&quot; page
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23162'>FLINK-23162</a>] -         Create table uses time_ltz in the column name and it&#39;s expression which results in exception 
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23168'>FLINK-23168</a>] -         Catalog shouldn&#39;t merge properties for alter DB operation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23178'>FLINK-23178</a>] -         Raise an error for writing stream data into partitioned hive tables without a partition committer
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23200'>FLINK-23200</a>] -         Correct grammatical mistakes in &#39;Table API&#39; page of &#39;Table API &amp; SQL&#39;
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23226'>FLINK-23226</a>] -         Flink Chinese doc learn-flink/etl transformation.svg display issue
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23312'>FLINK-23312</a>] -         Use -Dfast for building e2e tests on AZP
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-23315'>FLINK-23315</a>] -         Bump log4j to 2.14.1 for version 1.13.2
```

Comment:
We should remove this line since the JIRA is still not resolved and I've updated its fix version to 1.13.3 in JIRA.